mqtt/mqtt-bridge/src/token_source.rs

```diff
@@ -14,7 +13,7 @@ define_encode_set! {
 
 #[async_trait]
 pub trait TokenSource {
```

Comment:
Usually, it is better to make an associated type for Error. and then the implementor defines exact error type.